lib/coremget.go

```diff
@@ -45,18 +69,34 @@ func (c *Conn) MGet(index string, _type string, mgetRequest MGetRequestContainer
 	return retval, err
 }
 
+func mGetUrl(index string, _type string) string {
```

Comment:
same comment as the above.  Also, when you make these member functions it is a great time to write tests.  Please write tests.